src/server/routes/information.js

```diff
@@ -0,0 +1,79 @@
+/* eslint-disable no-unused-vars */
+
+const express = require('express')
+const swaggerJsdoc = require('swagger-jsdoc')
+
+const apiSpec = require('../api-spec.json')
+
+/**
+ * @param {express.Express} app
+ */
+const setup = (app) => {
+  /**
+   * @swagger
+   *  /api/health:
+   *    get:
+   *      description: Check if the API server is running and operational
+   *      tags:
+   *        - Information
+   *      produces:
+   *        - application/json
+   *      responses:
+   *        200:
+   *          description: Witness the firepower of this fully ARMED and OPERATIONAL battle station!
+   *          content:
+   *            application/json:
+   *              schema:
+   *                type: object
+   *                properties:
+   *                  message:
+   *                    type: string
+   *                    required: true
+   *        500:
+   *          description: It's a trap!
+   *          content:
+   *            application/json:
+   *              schema:
+   *                $ref: '#/components/schemas/Error'
+   */
+  app.get('/api/health', (_, res) => {
+    return res.status(200).json({
+      message: `
+      Once upon a midnight dreary, while I pondered, weak and weary,
```

Comment:
hahaha what

Reply:
It felt like a waste returning an empty response 😆